expense/tests.py

```diff
@@ -135,3 +137,51 @@ def test_addexpense(self):
         })
         
         self.assertNotEqual(response2.content,response.content)
+    
+class TotalExpense(TestCase):
+    def setup_account(self):
+        self.client.post(register_url,json.dumps({
+            'email' : 'test@test.com',
+            'name' : 'testwithdjango',
+            'password': 'test',
+        }),content_type=content_type)
+
+    def test_totalexpense(self):
```

Comment:
I think you can change this function to these to prevent duplication code
```suggestion
    def test_totalexpense(self):
        self.setup_account()
        session = self.client.session
        session['_auth_user_id'] = 'test@test.com'
        session.save()
        response = self.client.get('/expense/total-expense/',{
            'session_id': session.session_key,
        })

        self.assertEqual(response.status_code,200)
```